### ISSUE TITLE
merge upstream PRs

### ISSUE THREAD

**Issue by fossfreedom** [NicoHood/arc-theme#1]
Record of current PRs.  Edit and add comment as to whether candidate for merge, initial who is testing the merge

- [ ] https://github.com/horst3180/arc-theme/pull/800 "Fixed xfce4 gtk3 panel plugin menus " Merge Candidate
- [ ] https://github.com/horst3180/arc-theme/pull/790 add support for xed 
- [ ] https://github.com/horst3180/arc-theme/pull/788 remove margin:auto  Merge Candidate
- [ ] https://github.com/horst3180/arc-theme/pull/785 Tweak style for switching windows directly  Merge Candidate
- [ ] https://github.com/horst3180/arc-theme/pull/749 Fixing bold font in GTK3 XFCE4 Panel Plugin Menus  Merge Candidate
- [ ] https://github.com/horst3180/arc-theme/pull/739 Fix mixed quote types in gulpfile  Merge Candidate
- [ ] https://github.com/horst3180/arc-theme/pull/703 Window borders in Xfce  Merge Candidate
- [ ] https://github.com/horst3180/arc-theme/pull/676 add yarn.lock 
- [ ] https://github.com/horst3180/arc-theme/pull/656 Implement automatic precompilation of the theme 
- [ ] https://github.com/horst3180/arc-theme/pull/436 Install Gtk3 theme versioned 
- [ ] https://github.com/horst3180/arc-theme/pull/413 Added docky theme to match Merge Candidate

**Comment by NicoHood** [comment]
I was working on #800 and will open a new PR here once I've fixed everything. Unfortunately my internet is currently **extremely** slow, so I am unable to do anything. I think it is a good idea to always post a pre/after screenshot for better issue tracking, especially if horst may ever come back. I will try to review your PRs, if you open anything here :)

**Comment by fossfreedom** [comment]
@NicoHood re #785 - screenshot in that PR.  I couldnt get this working on gnome-shell 3.26.

I'm not sure I actually like the PR anyway - I think it ventures too far from adwaita.  Thoughts?

**Comment by NicoHood** [comment]
I agree. I would reject that PR. It does not look the arc way, nor does it look good to me.

**Comment by NicoHood** [comment]
I've added interesting issues from the first github page. Will comment on all other issues once the first people react on my comment. 150 issues is immense, we need to get rid of those and point people to the new upstream. Sadly we have no write access to close the issues...

**Comment by dglava** [comment]
Pull request #38 addresses https://github.com/horst3180/arc-theme/issues/836.

**Comment by NicoHood** [comment]
The fork is now detached from the horst repository.
